distros/kubernetes/nvsentinel/values-tilt.yaml

```diff
@@ -21,6 +21,8 @@ global:
   # - values-tilt-postgresql.yaml (when USE_POSTGRESQL=1)
   # This ensures proper configuration for both database backends
 
+  ctrlRuntimeEnabled: false
```

Comment:
once tests pass with this disabled I'll do another run with it enabled and leave it enabled